### ISSUE TITLE
App issue: Ghostfolio

### ISSUE THREAD

**Issue by zetix** [caprover/one-click-apps#1139]
READ FIRST:

### App bugs:

Unable to enable HTTPS after installing Ghostfolio. All associated docker containers are using the `latest` tag.

The error message when attempting to enable HTTPS: `1107 : Verification Failed.`

![image](https://github.com/user-attachments/assets/ded0187f-928b-49a8-868d-f131beb51ba2)

### Installation bugs:

No issues with installation.

Build log:

```log
------------------------- Sun Jul 28 2024 03:12:22 GMT+0000 (Coordinated Universal Time)
Build started for ghost
An explicit image name was provided (ghostfolio/ghostfolio:latest). Therefore, no build process is needed.
Pulling this image: ghostfolio/ghostfolio:latest This process might take a few minutes.
Build has finished successfully!
```

App log snippet:

```log
2024-07-28T03:18:02.632745004Z [Nest] 107  - 07/28/2024, 3:18:02 AM     LOG [NestApplication] Nest application successfully started +545ms
2024-07-28T03:18:02.639005485Z [Nest] 107  - 07/28/2024, 3:18:02 AM     LOG    ________               __  ____      ___
2024-07-28T03:18:02.639369756Z [Nest] 107  - 07/28/2024, 3:18:02 AM     LOG   / ____/ /_  ____  _____/ /_/ __/___  / (_)___
2024-07-28T03:18:02.639696283Z [Nest] 107  - 07/28/2024, 3:18:02 AM     LOG  / / __/ __ \/ __ \/ ___/ __/ /_/ __ \/ / / __ \
2024-07-28T03:18:02.639992185Z [Nest] 107  - 07/28/2024, 3:18:02 AM     LOG / /_/ / / / / /_/ (__  ) /_/ __/ /_/ / / / /_/ /
2024-07-28T03:18:02.640336319Z [Nest] 107  - 07/28/2024, 3:18:02 AM     LOG \____/_/ /_/\____/____/\__/_/  \____/_/_/\____/ 2.98.0
2024-07-28T03:18:02.640589270Z [Nest] 107  - 07/28/2024, 3:18:02 AM     LOG 
2024-07-28T03:18:02.640864532Z [Nest] 107  - 07/28/2024, 3:18:02 AM     LOG Listening at http://ghost.cap.domain.com:3333
2024-07-28T03:18:02.641105134Z [Nest] 107  - 07/28/2024, 3:18:02 AM     LOG 
```

**Comment by githubsaturn** [comment]
First of all, `latest` tag is unstable and [you should never use it](https://vsupalov.com/docker-latest-tag/#:~:text=You%20should%20avoid%20using%20the,apart%20from%20the%20image%20ID.). The one click apps are only tested with the default values that you find in the installer, changing the version may or may not work. For example if the one click app template is create with version 2.5 and version 2.6 introduces a new required env variable, the template wouldn't have that and hence it will fail. Chance of that happening for minor version upgrades is low and for major version upgrades (2.x to 3.x for example).



Having said all of this, `1107 : Verification Failed` is completely unrelated to one click app and it just means that your CapRover instance isn't able to verify the domain. You can follow this script to temporarily disable the domain verification:
https://caprover.com/docs/troubleshooting.html#domain-verification-failed---error-1107

**Comment by zetix** [comment]
Using `latest` doesn't appear to be the problem as all of the services installed and started up correctly. While I understand using `latest` might not be ideal, I was trying to see if using the latest version of the apps/services would resolve the verification issue in case it's version specific.

I have also tried disabling domain verification but I'm still getting the verification failed error.

This error also appears to only occur for Ghostfolio. Installing any other app and enabling HTTPS works just as expected. 

Just to confirm again, I installed Ghostfolio using the default configuration, and still faces the same issue despite disabling domain verification.

Default version:
![image](https://github.com/user-attachments/assets/49530f98-c560-4f11-9dc0-9fb06a40cc4f)
Enabling HTTPS:
![image](https://github.com/user-attachments/assets/4d4fe01e-baed-433b-b0e0-f05e1ab465b3)
